catroweb.yaml

```diff
@@ -1121,3 +1216,99 @@ components:
           type: number
           example: 0.14381762458251943
           description: filesize in megabytes
+
+    NotificationsId:
+      type: integer
+      minimum: 0
+      description: Id of a notification
+      default: 0
+      example: 2
+
+    NotificationsType:
+      type: string
+      example: reaction
+
+    NotificationsCountResponse:
+      type: object
+      properties:
+        all:
+          type: integer
+          example: 9
+        likes:
+          type: integer
+          example: 1
+        followers:
+          type: integer
+          example: 0
+        comments:
+          type: integer
+          example: 5
+        remixes:
+          type: integer
+          example: 3
```

Comment:
Please change `all` to `total`. Likes etc should be singular eg `likes` should be `like` .